src/xpublish_tiles/xpublish/tiles/plugin.py

```diff
@@ -501,9 +501,6 @@ async def get_dataset_tile(
                         format=query.f,
                     )
 
-            return StreamingResponse(
-                buffer,
-                media_type="image/png",
-            )
+            return Response(buffer, media_type="image/png")
```

Comment:
```suggestion
            return Response(buffer.getbuffer(), media_type="image/png")
```

This should give you a `memoryview` of the bytes, so no copy. We're throwing it away after the the request is done so no chance of weird side effects causing a problem.